wgpu-hal/src/gles/web.rs

```diff
@@ -171,6 +171,13 @@ impl Surface {
         ))?;
 
         if swapchain.format.describe().srgb {
+            // Important to set the viewport since we don't know in what state the user left it.
+            gl.viewport(
```

Comment:
Actually is scissor rect also a problem?

Reply:
nope, because there is a `gl.disable(glow::SCISSOR_TEST);` a few lines below 😄